com.Audentia.BlocNotes/com.Audentia.BlocNotes/DetailViewController.m

```diff
@@ -50,6 +51,26 @@ - (IBAction)shareNote:(id)sender {
     
 }
 
+    
+- (IBAction)didTapEditButton:(id)sender {
+
+    switch (self.noteText.editable) {
```

Comment:
Why don't you just use a regular if/else? A switch with YES and NO cases seems a bit unorthodox.